Source/EasyNetQ.IntegrationTests/PubSubTest.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace EasyNetQ.IntegrationTests
+{
+    public class IntegrationTestMessage
+    {
+    }
+
+    [Collection("RabbitMQ")]
+    public class PubSubTest: IDisposable
+    {
+        private readonly IBus bus;
+
+        public PubSubTest(RabbitMQFixture fixture)
+        {
+            bus = RabbitHutch.CreateBus($"host={fixture.Host}");
+        }
+
+        [Fact]
+        public async Task Should_be_able_to_publish_subscribe()
+        {
+            var subscriptionId = Guid.NewGuid().ToString();
+            var messageDeliveredEvent = new CountdownEvent(1);
+            using (bus.Subscribe<IntegrationTestMessage>(subscriptionId, x => messageDeliveredEvent.Signal()))
+            {
+                await bus.PublishAsync(new IntegrationTestMessage()).ConfigureAwait(false);
+                messageDeliveredEvent.Wait();
```

Comment:
```suggestion
                if (!messageDeliveredEvent.Wait(5000))
                    throw new TimeoutException();
```

If something goes wrong, then this test will at least not freeze.